google-map-search.html

```diff
@@ -123,6 +111,15 @@
         type: Array,
         value: function() { return []; },
         notify: true
+      },
+
+      /**
+       * The lat/lng location.
+       */
+      location: {
+        type: Object,
+        value: function() { return {}; },
```

Comment:
I believe this will break the `var location = this.location ? this.location : this.map.getCenter();` check in line 156